docs/02-app/02-api-reference/01-components/link.mdx

```diff
@@ -220,7 +220,9 @@ export default function Page() {
 Prefetching happens when a `<Link />` component enters the user's viewport (initially or through scroll). Next.js prefetches and loads the linked route (denoted by the `href`) and data in the background to improve the performance of client-side navigations. Prefetching is only enabled in production.
 
 - **`true` (default)**: The full route and its data will be prefetched.
-- `false`: Prefetching will be disabled.
+- `false`: Prefetching will be disabled. 
+
+> **Good to know**: When `prefetch` is set to `false`, prefetching will still occur on hover.
```

Comment:
Isn't prefetching will still occur on hover in development mode ?

Reply:
@karimxload It does not on either Pages or App Router in development mode.